mobile/src/main/java/com/google/samples/apps/iosched/ui/onboarding/WelcomePostConferenceFragment.kt

```diff
@@ -21,18 +21,25 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
 import com.google.samples.apps.iosched.R
 
 /**
  * First page of onboarding showing a welcome message post the conference.
  */
 class WelcomePostConferenceFragment : Fragment() {
 
+    private val viewModel: OnboardingViewModel by activityViewModels()
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         return inflater.inflate(R.layout.fragment_onboarding_welcome_post, container, false)
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        viewModel.reportFullyDrawn()
```

Comment:
OnboardingActivity doesn't gate `reportFullyDrawn()` with `doOnLayout` or similar, so I wonder if this will report too soon (the view is created, but is it actually shown yet?)

Reply:
I assumed that the view was already completed at this point since it is hard coded data in the view. Just to be safe, I am adding `doOnLayout` just in case there is still some measuring happening. I assume the difference in start up time would be negligible.